pyaedt/hfss.py

```diff
@@ -465,8 +464,8 @@ def create_linear_count_sweep(self, setupname, unit, freqstart, freqstop, num_of
         Examples
         --------
 
-        Create a setup named ``'LinearCountSetup'`` and use it in a linear count sweep
-        named ``'LinearCountSweep'``.
+        Create a setup named ``"LinearCountSetup"`` and use it in a linear count sweep
+        named ``"LinearCountSweep"``.
```

Comment:
@PipKat @akaszynski @MaxJPRey following my previous question, so we should follow the convention of using tick marks for object names in the example descriptions?

Reply:
Here it was obvious to me that it was an object and so I used the one back tick, per numpy doc.